macros/g4simulations/GlobalVariables.C

```diff
@@ -1,5 +1,32 @@
 #pragma once
-#if ROOT_VERSION_CODE >= ROOT_VERSION(6,00,0)
 static bool overlapcheck = false;
 static double no_overlapp = 0.0001; // added to radii to avoid overlapping volumes
-#endif
+
+namespace Enable
+{
+  static bool AEROGEL = false;
+  static bool BLACKHOLE = false;
+  static bool CEMC = false;
+  static bool DIRC = false;
+  static bool EEMC = false;
+  static bool FEMC = false;
+  static bool FHCAL = false;
+  static bool HCALIN = false;
+  static bool HCALOUT = false;
+  static bool MAGNET = false;
+  static bool OVERLAPCHECK = false;
```

Comment:
Is this boolean used? It appears the macros still calls `static bool overlapcheck = false` on line 2